src/main.rs

```diff
@@ -33,67 +25,151 @@ struct App {
+    Sign(SignCmd),
+    /// Export the private key of an account from its index.
+    ///
+    /// WARNING: This prints your account's private key to stdout!
+    ExportPrivateKey,
+}
+
+#[derive(Debug, Subcommand)]
+enum SignCmd {
+    /// Sign a transaction given it's ID.
+    Tx {
+        /// The transaction ID.
+        tx_id: fuel_types::Bytes32,
     },
 }
 
+const ABOUT: &str = "A forc plugin for generating or importing wallets using BIP39 phrases.";
+const EXAMPLES: &str = r#"
+EXAMPLES:
+    # Create a new wallet at the default path `~/.fuel/wallets/.wallet`.
+    forc wallet new
+
+    # Import a new wallet from a mnemonic phrase.
+    forc wallet import
+
+    # Derive a new account for the default wallet.
+    forc wallet account new
+
+    # Derive a new account for the wallet at the given path.
+    forc wallet --path /path/to/wallet account new
+
+    # Derive (or re-derive) the account at index 5.
+    forc wallet account 5 new
+
+    # Sign a transaction via its ID with account at index 3.
+    forc wallet account 3 sign tx 0x0bf34feb362608c4171c87115d4a6f63d1cdf4c49b963b464762329488f3ed4f
+
+    # Export the private key of the account at index 0.
+    forc wallet account 0 export-private-key
+"#;
+
 #[tokio::main]
 async fn main() -> Result<()> {
     let app = App::parse();
-    match app.command {
-        Command::New => new_account_cli(app.wallet_path)?,
-        Command::List => print_account_list(app.wallet_path)?,
-        Command::Init => init_wallet_cli(app.wallet_path)?,
-        Command::Account { index } => print_account_address(app.wallet_path, index)?,
-        Command::Sign { id, account_index } => {
-            sign_transaction_cli(&id, account_index, app.wallet_path)?
+    let wallet_path = app.wallet_path.unwrap_or_else(utils::default_wallet_path);
+    match app.cmd {
+        Command::New => new_wallet_cli(&wallet_path)?,
+        Command::Import => import_wallet_cli(&wallet_path)?,
+        Command::Accounts => account::print_address_list(&wallet_path)?,
+        Command::Account(Account { index, cmd }) => match (index, cmd) {
+            (None, Some(AccountCmd::New)) => account::new_cli(&wallet_path)?,
+            (Some(acc_ix), Some(AccountCmd::New)) => {
+                account::new_at_index_cli(&wallet_path, acc_ix)?
+            }
+            (Some(acc_ix), None) => account::print_address(&wallet_path, acc_ix)?,
+            (Some(acc_ix), Some(AccountCmd::Sign(SignCmd::Tx { tx_id }))) => {
+                sign_transaction_cli(&wallet_path, tx_id, acc_ix)?
+            }
+            (Some(acc_ix), Some(AccountCmd::ExportPrivateKey)) => {
+                account::export_cli(&wallet_path, acc_ix)?
+            }
+            (None, Some(cmd)) => print_account_subcmd_index_warning(&cmd),
+            (None, None) => print_account_subcmd_help(),
+        },
+        Command::SignPrivate(SignCmd::Tx { tx_id }) => {
+            sign_transaction_with_private_key_cli(tx_id)?
         }
```

Comment:
suggestion: move this code to a function like `account_cli` in a new file `src/account.rs`